Add units for on-site carbon capture

yalinli2 opened the pull request:
Added a _carbon_capture.py module for pre- and post-combustion CO2 capture, think this will be useful in the future. Simple examples included in the docstring, but will update the examples once it's used in a biorefinery. Passed pytest on my end. 😃

yoelcortes reviewed (changes requested):
Hi Yalin, excellent work! The documentation is nicely detailed, the tests ran well, and the coverage increased. I just have very minor corrections:

In the biosteam/evaluation/_model.py file, remove line 421 where "dropna()" is used.

In the the _carbon_capture module:
1. Remove "contributors" in line 4 and add your name and email. Note that these are module level copyrights.
2. Remove lines 9-16. I think the way it works is best.
3. In the documentation for AmineAbsorbtion, line 69-70, please remove the description for fermentation.

Also, please use better notes for the commits. For example, instead of "update", you can say "correct documentation for AmineAbsorption". This helps a bunch when looking back on the history for others using BioSTEAM. Sorry I didn't follow good practice before either...

Thank you!

yalinli2 commented:
Thanks for the comments! I've addressed as recommended, sorry that I introduced a bug when fixing _model.py so after fixing the bug, the most recent commit message is not about this PR...

Let me know anything else needed!

yoelcortes commented:
Perfect, thanks for adding the units!
